dashboard/app/models/code_review_group.rb

```diff
@@ -11,9 +11,11 @@
 # Indexes
 #
 #  index_code_review_groups_on_section_id  (section_id)
+#  TODO: Add index on follower_id
 #
 class CodeReviewGroup < ApplicationRecord
   # use dependent: :delete_all here because code_review_group_members is a join table and has no id column.
   has_many :members, class_name: 'CodeReviewGroupMember', dependent: :delete_all
+  has_many :followers, through: :members
```

Comment:
Could we name this something else so it's a little more clear what it is? What is a code review group follower?

Reply:
Any suggestions?

In general, I find the `follower` terminology pretty confusing throughout, but I think we tend to use it to mean "user in section".  I had tried `member_follower` but that seemed just as confusing.  I think the crux of the problem is that there are at least three models that correspond to the student in this context -- a CodeReviewGroupMember, Follower, and User.

I also wrote a version of this where I skipped follower entirely and a CodeReviewGroup just had users but that unnecessarily joined against the users table for this scenario.  I can try that again if you think that would be better.

Reply:
It's functionally a little bit different in that the `map` will result in an n+1 query but the association will result in a single query with an INNER JOIN.  Removing the n+1 query was my primary motivation for adding this association but I think it could also help with readability (if we had a better shared understanding of what a `Follower` represents or a better name).

Reply:
I did not know that was a benefit of using `has_many`, TIL!